Sources/Identity/CustomerInfoManager.swift

```diff
@@ -376,6 +374,24 @@ private extension CustomerInfoManager {
                                      completion: completion)
     }
 
+    /// Posts all `transactions` in parallel.
+    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    private func postTransactions(
+        _ transactions: [StoreTransaction],
+        _ data: PurchasedTransactionData
+    ) async {
+        await withTaskGroup(of: Void.self) { group in
+            for transaction in transactions {
+                group.addTask {
+                    _ = await self.transactionPoster.handlePurchasedTransaction(
```

Comment:
should we have a log for these?

Reply:
`TransactionPoster` already has a log in `handlePurchasedTransaction`.